stencil-library/src/components/payouts-list/payouts-list.tsx

```diff
@@ -98,22 +97,30 @@ export class PayoutsList {
     }
     this.loading = true;
 
-    const api = Api(this.authToken, config.proxyApiOrigin);
-    const endpoint = `account/${this.accountId}/payouts`;
+    try {
+      const api = Api(this.authToken, config.proxyApiOrigin);
+      const endpoint = `account/${this.accountId}/payouts`;
 
-    const response: IApiResponseCollection<IPayout[]> = await api.get(endpoint, this.params);
-    if (!response.error) {
-      this.paging = {
-        ...this.paging,
-        ...response.page_info
-      }
+      const response: IApiResponseCollection<Payout[]> = await api.get(endpoint, this.params);
```

Comment:
Similarly - I think this one should still be `IPayout` - since its the raw data before being instantiated - right?

Reply:
@colganmeanor yes, I think you're right - `IPayout` represents the data that comes from the server before we call `new Payment()` to add things like computed properties

Reply:
Sorry guys, I oversight this while merging. I am adding it back